src/raftstore/store/peer.rs

```diff
@@ -436,7 +437,6 @@ impl Peer {
         } else {
             initialized
         };
-        self.pending_remove = true;
```

Comment:
Why remove the mark?

Reply:
I move this mark to `handle_destroy_peer`. 
It's more clear to set this mark before really doing destroy things.
The `maybe_destroy` function should only check whether it is valid to destroy and change nothing.